packages/ui-react/src/containers/PaymentContainer/PaymentContainer.tsx

```diff
@@ -44,6 +44,13 @@ const processBillingReceipt = (receipt: Blob | string, transactionId: string) =>
   }
 };
 
+const EXTERNAL_PAYMENT_METHODS = ['Apple In-App', 'Android In-App', 'Roku In-App'];
+const STORE_LINKS: Record<string, string> = {
+  apple: 'https://support.apple.com/en-qa/118428',
+  android: 'https://support.google.com/googleplay/answer/7018481?hl=en',
+  roku: 'https://support.roku.com/article/208756478',
```

Comment:
I don't want to overcomplicate things; but somewhere in the future we need to support multilingual links

Reply:
Good point. I tested it without language parameters, and this seems to redirect you to the correct language. I think this is the best solution instead that we choose the language.

Reply:
I removed the language parameters and made a fix for Android (Google) IAP payments not being recognized.

Reply:
It doesn't redirect me to the correct language when I set my browser to German using this method: https://stackoverflow.com/a/41190151

After a small research, these are the correct (auto redirect) URLs:

Apple: https://support.apple.com/118428
Android: https://support.google.com/googleplay/answer/7018481 (removed the `hl` query string)
Roku: unknown. But these are mostly targetted to US audience, but I consider this less a problem.